images/bootstrap/Dockerfile

```diff
@@ -2,4 +2,4 @@ FROM ubuntu:14.04
 
 ADD stolonboot /stolonboot
 
-CMD ["/stolonboot", "-proxies", "2", "-sentinels", "2"]
+CMD ["/stolonboot", "-proxies", "1", "-sentinels", "2"]
```

Comment:
It needed for success of install hook.
Install job create pod which dies with `CrashLoopBackOff` and logs that resources already created.

I've spent some time to find out reasons for first crash, it's
```
time="2016-07-08T10:28:04Z" level=info msg="cmd output: replicationcontroller \"stolon-proxy\" scaled\n" 
time="2016-07-08T10:28:04Z" level=info msg="looking for 2 pods, have 2 pods, 1 healthy"
```

Because 1 pod can't be scheduled
```
NAMESPACE     NAME                              READY     STATUS             RESTARTS   AGE
default       stolon-proxy-c0j7d                1/1       Running            0          2m
default       stolon-proxy-xhxsv                0/1       Pending            0          2m
```

So we need some labels refactoring or more nodes. By now I prefer to have 2 keepers instead and scale down proxy to 1.